src/main/java/fire/department/Call.java

```diff
@@ -0,0 +1,23 @@
+package fire.department;
+
+public class Call {
```

Comment:
why are you creating a class with one field inside?

Reply:
shell I replace it by String?